### PR TITLE
fix rally assets

### PR DESCRIPTION
**Pull request by aspacca**
Enhancement
## Proposed commit message

Fix timestamp range from/to for kubernetes integrations
Remove event.ingested for aws ec2logs

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
~- [ ] I have added an entry to my package's `changelog.yml` file.~
~- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).~

## Author's Checklist

- [ ] 

## Related issues

-

### CODE REVIEW

File: packages/kubernetes/_dev/benchmark/rally/container-benchmark/template.ndjson [hunk 1]
```diff
@@ -1,15 +1,15 @@
-{{- $period := generate "metricset.period" }}
-{{- $agentId := generate "agent.id" }}
-{{- $agentVersion := generate "agent.version" }}
-{{- $agentName := generate "agent.name" }}
-{{- $agentEphemeralid := generate "agent.ephemeral_id" }}
+{{- $period := generate "metricset.period" -}}
```
Comment (ruflin) [inline packages/kubernetes/_dev/benchmark/rally/container-benchmark/template.ndjson:1]: What is the `-` at the end doing?

File: packages/kubernetes/_dev/benchmark/rally/container-benchmark/template.ndjson [hunk 2]
```diff
@@ -145,82 +145,83 @@
          "deployment":{
             "name":"demo-deployment-{{ $rangeofid }}"
          }
+      }
+   },
+   "cloud": {
+      "provider": "gcp",
+      "availability_zone": "europe-west1-d",
+      "instance":{
+      "name":  "{{ $agentName }}" ,
```
Comment (ruflin) [inline packages/kubernetes/_dev/benchmark/rally/container-benchmark/template.ndjson:154]: Nit
```suggestion
      "name": "{{ $agentName }}",
```

**Review by ruflin — APPROVED** [review]

File: packages/kubernetes/_dev/benchmark/rally/container-benchmark/template.ndjson [hunk 3]
```diff
@@ -145,82 +145,83 @@
          "deployment":{
             "name":"demo-deployment-{{ $rangeofid }}"
          }
+      }
+   },
+   "cloud": {
```
Comment (gizas) [inline packages/kubernetes/_dev/benchmark/rally/container-benchmark/template.ndjson:150]: @aspacca I have tested and the cloud section is needed both in container and pod when data come from Cloud providers. So in example we have hardocded gcp and is ok

File: packages/kubernetes/_dev/benchmark/rally/container-benchmark/template.ndjson (hunk 1 above)
Comment (gizas) [reply packages/kubernetes/_dev/benchmark/rally/container-benchmark/template.ndjson:1]: does not add newline

File: packages/kubernetes/_dev/benchmark/rally/pod-benchmark/template.ndjson [hunk 4]
```diff
@@ -151,7 +151,7 @@
       "type":"kubernetes"
    },
    "data_stream":{
-      "namespace":"default",
+      "namespace":"ep",
```
Comment (gizas) [inline packages/kubernetes/_dev/benchmark/rally/pod-benchmark/template.ndjson:154]: Why is this changed the namespace?

**Comment by gizas** [comment]
LGTM apart from the question regarding: https://github.com/elastic/integrations/pull/8654/files#r1415715491

File: packages/kubernetes/_dev/benchmark/rally/container-benchmark/template.ndjson [hunk 5]
```diff
@@ -145,82 +145,83 @@
...
+         "name":"kubernetes-scale",
+         "url":"https://{{ generate "Ip" }}"
       }
+   },
+   "service":{
+      "address": "https://{{ $agentName }}:10250/stats/summary",
+      "type":"kubernetes"
+   },
+   "data_stream":{
+      "namespace":"ep",
```
Comment (gizas) [inline packages/kubernetes/_dev/benchmark/rally/container-benchmark/template.ndjson:181]: Also namespace here why chnaged?

**Review by gizas — APPROVED** [review]

File: packages/kubernetes/_dev/benchmark/rally/pod-benchmark/template.ndjson (hunk 4 above)
Comment (aspacca) [reply packages/kubernetes/_dev/benchmark/rally/pod-benchmark/template.ndjson:154]: because in `elastic-package` benchmark the datastream is with the "ep" namespace

Comment (ruflin) [reply packages/kubernetes/_dev/benchmark/rally/pod-benchmark/template.ndjson:154]: @gizas Will this create a problem for the rally benchmark?

**Review by tommyers-elastic — APPROVED** [review]
